test/integration/data-fetching-errors/test/index.test.js

```diff
@@ -91,34 +88,6 @@ const runTests = (isDev = false) => {
       `getStaticPaths can not be attached to a page's component and must be exported from the page`
     )
   })
-
```

Comment:
Existing tests should not be changed.

Reply:
I added the tests back but modified them to reflect the new changes